docs/serverless/08-01-create-function.md

```diff
@@ -56,8 +56,8 @@ Follows these steps:
 
     </details>
     <details>
-    <summary label="console-ui">
-    Console UI
+    <summary label="ui">
+    UI
```

Comment:
```suggestion
    Console UI
```
Refer to [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478304363).